src/components/api-reference/schemas/measurements.js

```diff
@@ -18,7 +18,12 @@ const BOOLEAN_FIELDS = [
 
 const PROPORTION_FIELDS = [
   {name: 'numerator', value: 'integer', description: 'The number of occurrences for which the measure criteria are satisfied. Must be greater than or equal to zero and less than or equal to the <b>denominator</b>.', notes: 'writable'},
-  {name: 'denominator', value: 'integer', description: 'The total number of occurrences as described by the measure. Must be greater than or equal to zero.', notes: 'writable'}
+  {name: 'denominator', value: 'integer', description: 'The total number of occurrences as described by the measure. Must be greater than or equal to zero. Can only be 0 if the numerator is 0 as well.', notes: 'writable'}
+];
+
+const CONTINUOUS_VARIABLE_FIELDS = [
+  {name: 'numerator', value: 'float', description: 'Must be greater than or equal to zero.', notes: 'writable'},
```

Comment:
What should I include as the first sentence of the description for numerator and denominator? I asked @glgaudet if "Can be anything" was appropriate and she suggested noting that it varies, is defined by the submitting org, and including a link to where CMS gives guidance on CV measurements. 

Wdyt? @marimiyachi @ivanang @kencheeto

Reply:
+1 to @glgaudet 's advice, that seems helpful